email/1.3/modules/ROOT/edits/email-list.adoc

```diff
@@ -0,0 +1,210 @@
+= List Emails with Email Connector Examples - Mule 4
+:page-aliases: connectors::email/email-list.adoc
+
+Anypoint for Email (Email Connector) provides the *List - IMAP* and *List - POP3* operations to list emails from any part of the flow. The following examples illustrate how to configure these operations for the connector:
+
+* <<send-attachments,List Emails from an IMAPS Server>> +
+Configure the *List - IMAP* operation to list all the emails in the configured IMAP mailbox folder from an IMAPS server that has a TLS connection. Additionally, set a *For Each* scope component and two *Set Variable* components to iterate over the emails and save the email body and attachment content in variables.
```

Comment:
ditto here: these should not be in bold unless referred to in the context of viewing and manipulating the interface controls that have those labels
```suggestion
Configure the List - IMAP operation to list all the emails in the configured IMAP mailbox folder from an IMAPS server that has a TLS connection. Additionally, set a For Each scope component and two Set Variable components to iterate over the emails and save the email body and attachment content in variables.
```
For example, you would bold these in a context like the following:
. In *List - IMAP*, configure the operation to list....
or
. In the *List - IMAP* section, configure an operation to list....

Does this make sense? Should I provide more info about naming things in general contexts vs. quoting the UI?